elephant/asset.py

```diff
@@ -457,37 +442,38 @@ def mask_matrices(matrices, thresholds):
 
 def _stretched_metric_2d(x, y, stretch, ref_angle):
     r"""
-    Given a list of points on the real plane, identified by their absciss x
-    and ordinate y, compute a stretched transformation of the Euclidean
+    Given a list of points on the real plane, identified by their absciss `x`
```

Comment:
Shouldn't it be "abscissa" not "absciss"? At least accoring to a quick check in a dictionary.

Reply:
I'll rename it to abscissa